README.md

```diff
@@ -1,5 +1,5 @@
-Library Institutions and Persons on GitHub
-==========================================
+Library Institutions and People on GitHub
```

Comment:
Why is this better?

Reply:
I'd refer to https://www.merriam-webster.com/words-at-play/people-vs-persons:

> However, in the last few decades the trend has been very much in favor of people, rather than persons. The AP Stylebook flatly states “The word people is preferred to persons in all plural uses.” And The New York Times Manual of Style and Usage now also states “Use people as the plural of person,” although they add that persons may still be used in certain settings (“letters to the editor, in untranslated texts, in direct quotations and in a few established idioms like displaced persons and missing persons bureau”).